defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# app_name: null   <= must set
+# app_user: null   <= must set
+
+app_ensure_user: true
+
+app_group: "{{ ansible_common_remote_group | default(app_user) }}"
+
+global_app_root: "/applications"
+app_root: "{{ global_app_root }}/{{ app_name }}"
+app_module: "{{ app_name }}"
+app_data_dir: "{{ app_root }}/data"
+app_dir: "{{ app_root }}/app"
+
+app_code_dir: "{{ app_dir }}/code"
+# app_repo: null   <= must set
+# app_tag: null    <= must set
+
+conda_isolate: false
+conda_base: "{{ app_dir if conda_isolate else global_app_root }}/miniconda3"
+conda_run: "{{ conda_base }}/bin/conda run --no-capture-output -n {{ app_name }}"
```

Comment:
I would put `conda_run` in `vars/main.yml` instead of `defaults/main.yml` since, to my understanding, it is not meant to be overridden.